docs_versioned_docs/version-ros2jazzy/components/yaml/attachments/a300/amp_sensor_arch.mdx

```diff
@@ -0,0 +1,35 @@
+<table>
+<tr>
+<td>
+<center>
+<figure>
+    <img src={require("./img/amp_sensor_arch.png").default} width="250" />
+</figure>
+</center>
+</td>
+<td>
+
+```yaml
+platform:
+  attachments:
+    - name: sensor_arch
+      type: a300.amp_sensor_arch
+      model: default
+      parent: enclosure_antenna_mount
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+      enabled: true
+```
+</td>
+</tr>
+</table>
+
+The AMP sensor arch includes the following links to attach sensors and other accessories:
+- `${name}_front_camera_mount`: located under the centre-top of the arch, this is the default
+  location for the front-facing teleoperation camera
+- `${name}_rear_camera_mount`: located under the centre-top of the arch, this is the default
+  location for the rear-facing teleoperation camera
+- `${name}_lidar_mount`: default mounting location for a secondary 3D lidar. This mount is only
+  usable when an additional bracket is physically installed on the arch.
+- `${name}_left_antenna_mount`: the default mounting location for the GNSS/INS sensor's left antenna
+- `${name}_right_antenna_mount`: the default mounting location for the GNSS/INS sensor's right antenna
```

Comment:
Add a blank line to the end ot MDX files, to avoid off build / render issues.

Reply:
This file already has a newline at the end of it.